cmd/kube-version-change/version.go

```diff
@@ -28,6 +28,8 @@ import (
 	"runtime"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1"
```

Comment:
api/latest already does that.
It also requires that apiVersion/register.go's init() is called.